package.json

```diff
@@ -49,7 +49,7 @@
     "svg-inline-loader": "0.8.2",
     "svg-url-loader": "6.0.0",
     "tar": "4.4.8",
-    "terser-webpack-plugin": "2.3.8",
+    "terser-webpack-plugin": "4.2.3",
```

Comment:
Only go to v4 - v5 drops support for webpack@4

Reply:
https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/CHANGELOG.md#500-2020-10-14